data/json/monsters/fish.json

```
@@ -700,7 +703,7 @@
     "type": "MONSTER",
     "name": { "str": "giant carp" },
     "description": "This thing seems like a carp, only swollen and very very angry.  Death is the gift of the carp god.",
-    "default_faction": "carp",
+    "default_faction": "mutant",
```

```suggestion
    "default_faction": "aquatic_predator",
```
Provided that aquatic_predator is "by_mood" toward each other.